openapi.yml

```diff
@@ -1,8 +1,31 @@
 openapi: "3.0.0"
 info:
   title: "IO onboarding PA API"
-  description: "The backend used by the onboarding portal for public administrations of the IO project."
+  x-summary: >-
+    Public Administriations onboarding APIs for the IO project.
+  description: |-
+      The backend used by the onboarding portal for public administrations of the IO project.
+
+      The website for the project is [io-website].
+      Privacy Policy is [privacy].
+
+      Development happens on github.
+
+
+      [//]:  # (Riferimenti e url)
```

Comment:
I would write it in English

Reply:
Like above, it's just a placeholder to show what you can do with `description`. Feel free to use or remove that section. Examples of description are:

- https://developers.italia.it/it/api/siope.html
- https://developers.italia.it/it/api/muoversi-in-lombardia

@LucaPrete @alexgpeppe

Reply:
I don't know if such section is needed... Maybe @mccalv or @paomico can say something about it?